### DISCUSSION TITLE
PIP or PEP

### DISCUSSION THREAD

**Discussion by kehiy** [pactus-project/pactus#580] [Ideas]
I think we need a `Pactus enhancement proposal` or `Pactus improvement proposal`  repository in the `Pactus-project` (maybe we should add it in `Mainnet` launch?).

what do you think?
@b00f

**Comment by b00f** [comment]
I agree with the idea that we need a repository for a Pactus Enhancement Proposal or a Pactus Improvement Proposal within the Pactus project. It would be ideal if this could be managed by Pactus Lab. In the near future, we will have this Non-Profit Organization in place to make decisions about such matters

**Reply by kehiy** [reply]
very nice, why we don't make this repo in `pactus-project`?

**Reply by b00f** [reply]
Not sure. Let's be patent and give time:
“The strongest of all warriors are these two; Time and Patience.”

**Reply by kehiy** [reply]
sure!
we should wait and think about it.

**Reply by ja7ad** [reply]
@b00f Need to speak with Dr. alahbakhsh for PIP and Labs
